SwinUNETR/BTCV/requirements.txt

```diff
@@ -0,0 +1,6 @@
+git+https://github.com/Project-MONAI/MONAI#egg=monai

```

Comment:
could you pin down to a specific version see https://stackoverflow.com/a/13754517

Reply:
> could you pin down to a specific version see https://stackoverflow.com/a/13754517

Thanks. Addressed.